parachain/Cargo.toml

```diff
@@ -271,12 +271,12 @@ substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk", b
 
 # local
 core-primitives = { path = "../common/primitives/core", default-features = false }
-litentry-macros = { path = "../common/primitives/core/macros" }
-litentry-proc-macros = { path = "../common/primitives/core/proc-macros" }
-litentry-hex-utils = { path = "../common/utils/hex", default-features = false }
+heima-macros = { path = "../common/primitives/core/macros" }
+heima-proc-macros = { path = "../common/primitives/core/proc-macros" }
+heima-hex-utils = { path = "../common/utils/hex", default-features = false }
 runtime-common = { path = "runtime/common", default-features = false }
-litentry-parachain-runtime = { path = "runtime/litentry", default-features = false }
-paseo-parachain-runtime = { path = "runtime/paseo", default-features = false }
+heima-runtime = { path = "runtime/heima", default-features = false }
```

Comment:
`heima-node-runtime` ?

Reply:
I'll keep it as `heima-runtime` tho - `heima` actually stands for `litentry-parachain` already

`heima-node` is the node binary "of heima"